CHANGES.md

```diff
@@ -1,3 +1,7 @@
+# 0.2.4 (2025-09-16)
+- [UPGRADED] Spring Boot compilation version to `3.5.4`.
```

Comment:
```suggestion
- [UPGRADED] Spring Boot compilation version to `3.5.5`.
```

Reply:
I abhor the manual nature of our releases. Thanks for noticing, fixed, rebased and pushed.